cmd/kubeadm/app/apis/bootstraptoken/v1/utils.go

```diff
@@ -44,7 +44,7 @@ const (
 )
 
 // MarshalJSON implements the json.Marshaler interface.
-func (bts BootstrapTokenString) MarshalJSON() ([]byte, error) {
+func (bts *BootstrapTokenString) MarshalJSON() ([]byte, error) {
```

Comment:
Struct BootstrapTokenString has methods on both value and pointer receivers. Such usage is not recommended by the Go Documentation.